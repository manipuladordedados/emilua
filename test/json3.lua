local json = require('json')

json.decode('  ')
