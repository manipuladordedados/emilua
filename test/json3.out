Main fiber from VM 0x1 panicked: 'algorithm used requires more tokens than available'
stack traceback:
	input.lua:3: in main chunk
	[C]: in function ''
	[string "?"]: in function <[string "?"]:0>
